android/build.gradle

```diff
@@ -1,6 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
+
+	applicationVariants.all { variant ->
+        variant.getRuntimeConfiguration().exclude group: 'com.google.code.findbugs', module: 'jsr305'
+        variant.getRuntimeConfiguration().exclude group: 'org.nd4j', module: 'protobuf'
+    }
```

Comment:
ACRA needed guava as its dependency and there is lot of duplication between guava and nd4j which causes build failure. This is to avoid that.

Though I didn't needed to do this for a clean test app. So you could try building the app without this first.